cmd/ak/cmd/connections/delete.go

```diff
@@ -20,23 +18,16 @@ var deleteCmd = common.StandardCommand(&cobra.Command{
 		r := resolver.Resolver{Client: common.Client()}
 		c, id, err := r.ConnectionNameOrID(args[0])
 		if err != nil {
-			if errors.As(err, resolver.NotFoundErrorType) {
-				err = common.NewExitCodeError(common.NotFoundExitCode, err)
-			}
-			return err
+			return common.ToExitCodeError(err, "connection")
 		}
 		if !c.IsValid() {
-			err = errors.New("connection not found")
-			return common.NewExitCodeError(common.NotFoundExitCode, err)
+			return common.ToExitCodeError(sdkerrors.ErrNotFound, "connection")
 		}
 
 		ctx, cancel := common.LimitedContext()
 		defer cancel()
 
-		if err = connections().Delete(ctx, id); err != nil {
-			return fmt.Errorf("delete connection: %w", err)
-		}
-
-		return nil
+		err = connections().Delete(ctx, id)
+		return common.ToExitCodeError(err, "delete connection")
```

Comment:
this make me read it as "there is an error, convert it to exit code"
which is kind of confusing

Reply:
plz suggest better name?
we already have for example translateError which doing the same (converts nil to nil of there is no error)
I don't think we shoudl split it to if/else, but better name could do the job

Reply:
standard stuff, 
check if there is an error use common.ToExitCodeError(err, 'delete connection')
otherwise return null